<launch>
    <node pkg="phoxi_camera" type="phoxi_camera" name="phoxi_camera" output="screen"/>
    <node pkg="phoxi_camera" type="phoxi_camera_example.py" name="phoxi_camera_example" output="screen"/>
</launch>
